lesson9/stars_as_array.rb

```diff
@@ -0,0 +1,12 @@
+puts 'Сколько вам звезд на погоны?'
+user_input = gets.chomp.to_i
+def stars number
+  count = 1
+  shoulder_straps = ""
+  while count <= number do
+    shoulder_straps += '*'
+      count += 1
```

Comment:
remove 2 extra spaces here. 'count' shpuld be on the same indentation level as shoulder_straps